ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/PluginStatusDialog.java

```diff
@@ -153,16 +159,14 @@ private void createLink(Composite parent, int yesId, String pluginStatusDialog_v
 		ILaunchManager manager = DebugPlugin.getDefault().getLaunchManager();
```

Comment:
No other than the parent parameter is used within this method.
Please remove all unused parameters from `createLink()`.